ArraysHomework/ArraysHomework.cs

```diff
@@ -34,17 +37,18 @@ public void Arrays()
 
 
             // what wrong with this array declaration? Fix it and make it compile
-            string stringArray = new string[5];
+            string[] stringArray = new string[5];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray = new bool[];
+            bool[] boolArray = new bool[22];
 
             // What about this one? Fix it and make it compile
-            bool[] boolArray2 = new [20];
+            bool[] boolArray2 = new bool[20];
 
             // Take this array and copy its contents into a new array 
             char[] copyMe = { 'a', 'b', 'c' };
             char[] copiedArray = new char[3];
+            copiedArray = copyMe;
```

Comment:
We haven't talked much about references, but this creates a reference to the original array, so copyMe and copiedArray both refer to the same item in memory.
Don't use `copiedArray = copyMe`. Assign to the elements in copiedArray.